tasks/loadFixtures.js

```diff
@@ -0,0 +1,34 @@
+var _ = require('lodash');
+var request = require('request');
```

Comment:
Could you use [wreck](https://github.com/hapijs/wreck) here? I plan to transition us to it completely at some point